records_mover/db/driver.py

```diff
@@ -273,3 +187,14 @@ def tweak_records_schema_for_load(self,
                                       records_schema: RecordsSchema,
                                       records_format: BaseRecordsFormat) -> RecordsSchema:
         return records_schema
+
+
+class GenericDBDriver(DBDriver):
+    def loader_from_fileobj(self) -> None:
+        return None
+
+    def loader(self) -> None:
+        return None
+
+    def unloader(self) -> Optional[Unloader]:
+        return None
```

Comment:
I made a separate concrete class so implementers of new DBDriver subclasses make a conscious decision on bulk loading/unloading.